source/expressions.tex

```diff
@@ -3423,9 +3674,21 @@
 The \grammarterm{postfix-expression} is sequenced before
 each \grammarterm{expression} in the \grammarterm{expression-list}
 and any default argument.
-The initialization of a parameter,
+The initialization of a parameter or,  %JMB: do we need a comma before the or too?
```

Comment:
No, it's "X or (if Z) Y, ...."  where we use commas instead of parens.  But the or is just two-prong, thus no comma.

Reply:
ok